gradle/base/dsl-extensions/build.gradle.kts

```diff
@@ -1,3 +1,14 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget.JVM_21
+
 plugins {
     `kotlin-dsl`
 }
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_21
+    targetCompatibility = JavaVersion.VERSION_21
+}
+
+kotlin {
+    compilerOptions.jvmTarget = JVM_21
```

Comment:
Add xjdk-releaseas well

Reply:
Thanks for the pointer! I didn't know about this flag and have added it now.